grails-app/services/org/pih/warehouse/core/DocumentTemplateService.groovy

```diff
@@ -59,35 +60,37 @@ class DocumentTemplateService {
     }
 
     def renderOrderDocumentTemplate(Document documentTemplate, Order orderInstance, ConverterTypeTo targetDocumentType, OutputStream outputStream) {
+        try {
```

Comment:
Since the only changes here are lines 63 and 88-90, is there a way to get the code inside the try/catch to be ignored. I just want to avoid having this cause a conflict during the OBGM rebase.

Reply:
Nothing immediately comes to mind apart from doing `git diff -w` when resolving conflicts.